### ISSUE TITLE
User may have custom scripts -from URL or local file- that he wants to execute

### ISSUE THREAD

**Issue by bugra-derre** [chaostoolkit-incubator/chaostoolkit-azure#106]
The chaostoolkit-azure driver has scripts that can be executed on virtual machines and instances. Those script generates some noise on CPU, IO, disk and so on.

As this feature and capability is nice - it would be even nicer to allow a user to execute custom/ other scripts that he prepared. For example the scripts could be provided via his/her private GitHub repository, a public available script repository or on his private local file system.

@Lawouach Do you have oppinions on this?

**Comment by xpdable** [comment]
Definitely, I have PR similar features in aws and saltstack extension: https://github.com/chaostoolkit-incubator/chaostoolkit-aws/pull/73
https://github.com/chaostoolkit-incubator/chaostoolkit-saltstack/pull/7/files#diff-1a81de3bbab1a842459f2aa2600adda2R408

Also, months before @Lawouach memtions that might need to discuss whether we need to manage all script related function in a univeral way, but seems we are blocked for a long time by the virus :P  -> I can not find the issue already :-(
